example/README.md

```diff
@@ -0,0 +1,37 @@
+# When creating a new provider module
+
+Start with creating a folder under the provider-folder with the provider name.
+
+In [this folder](./example-game-provider/) you will find `example-game-provider` with some basic building blocks for creating a new provider module.
+- `router.go` - set up provider wallet api as well as endpoints operator will call, such as gamelaunch
+- `controller.go` - request parsing and validation from wallet calls
+- `wallet_service.go` - mainly a mapping service converting the provider specific api to work with the Valkyrie PAM api
+- `provider_service.go` - in the other way the Valkyrie operator api needs to be converted to work with the provider specifics, such as gamelaunch
+- `config.go` - define auth and provider specific from the configuration.
```

Comment:
Something about this sentence is a bit off, maybe a word missing?

Reply:
Referring to the keys in config. Will make it more clear